integrations/jenkins/README.md

```diff
@@ -9,6 +9,7 @@ Please follow the regular [installation instructions](https://www.jenkins.io/doc
 * [Docker Pipeline](https://plugins.jenkins.io/docker-workflow)
 * [File Parameter](https://plugins.jenkins.io/file-parameters/)
 * [Pipeline Utility Steps](https://plugins.jenkins.io/pipeline-utility-steps)
+* [Copy Artifact](https://plugins.jenkins.io/copyartifact/)
```

Comment:
Nit: For a better overview, please keep bullet items sorted alphabetically and make this go first.

Reply:
Moved the item to the top.